platforms/java/getJavaSdk.sh

```diff
@@ -46,6 +47,20 @@ downloadJavaSdk()
     IFS='.' read -ra VERSION_PARTS <<< "$JDK_VERSION"
     majorVersion="${VERSION_PARTS[0]}"
 
+    # Version 17 has a different url format than rest of the versions, so special casing it.
+    if [ "$majorVersion" == "17" ]; then
+        local buildNumber="12"
+        local url="https://download.java.net/java/GA/jdk${JDK_VERSION}/2a2082e5a09d4267845be086888add4f/${buildNumber}/GPL/openjdk-${JDK_VERSION}_linux-x64_bin.tar.gz"
```

Comment:
Do you know if this URL (specifically the `2a2082e5a09d4267845be086888add4f` part) will follow the same pattern for later versions of JDK 17? It may be something that we need to keep an eye out for when version 17.0.2 (or greater) of the JDK is released.

Reply:
That's a good point. I removed the URL parameterization and made update minor updates